src/query/combine.rs

```diff
@@ -119,11 +119,16 @@ where
     F: EntityTrait,
 {
     pub(crate) fn new(query: SelectStatement) -> Self {
+        Self::new_without_prepare(query)
+            .prepare_select()
+            .prepare_order_by()
+    }
+
+    pub(crate) fn new_without_prepare(query: SelectStatement) -> Self {
         Self {
             query,
             entity: PhantomData,
         }
-        .prepare_select()
         .prepare_order_by()
     }
```

Comment:
Did we invoke `prepare_order_by()` method twice? When we call `SelectTwoMany::new()` it will invoke `prepare_order_by()` method twice.